### PR TITLE
Pre prod

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

**Comment by martingartonft** [comment]
merge issues by the look of it?

File: services.yaml [hunk 1]
```diff
@@ -124,4 +123,8 @@ services:
 - name: wordpress-article-transformer@.service
   version: v38
   count: 2
-- name: zookeeper.service
\ No newline at end of file
+- name: zookeeper.service
+- name: neo4j@.service
+  count: 1
+- name: neo4j-sidekick@.service
+  count: 1
```
Comment (martingartonft) [inline services.yaml]: not in alpha order (run the script?)

File: neo4j@.service [hunk 2]
```diff
@@ -20,4 +20,4 @@ Restart=on-failure
 RestartSec=60
 
 [X-Fleet]
-MachineMetadata=persistent_tag=%i
+MachineMetadata=persistent_tag=2
```
Comment (martingartonft) [inline neo4j@.service]: like it!

File: neo4j-sidekick@.service [hunk 3]
```diff
@@ -5,8 +5,10 @@ After=neo4j@%i.service
 
 [Service]
 ExecStart=/bin/sh -c "\
+  etcdctl mkdir /ft/services/neo4j/servers; \
+  etcdctl set /vulcand/frontends/neo4j-db/frontend '{\"Type\": \"http\", \"BackendId\": \"vcb-neo4j\", \"Route\": \"PathRegexp(`/db/.*`) \"}'; \
```
Comment (martingartonft) [inline neo4j-sidekick@.service]: where is the backend created?

**Comment by catalin-ilea** [comment]
Yes, should be resolved now

File: neo4j-sidekick@.service (hunk 3 above)
Comment (tamas-molnar) [inline neo4j-sidekick@.service]: The backend is created by the VCB.

Comment (martingartonft) [inline neo4j-sidekick@.service]: of course, sorry.
so this front end is just for additionally routing on /db/*  cool.
thanks
+1

**Pull request merged**